sdk/python/kubeflow/training/utils/utils.py

```diff
@@ -210,6 +213,7 @@ def get_container_spec(
         command=command,
         args=args,
         volume_mounts=volume_mounts,
+        env=env,
```

Comment:
I think, you should convert env_vars to the appropriate models, like we do in Katib: https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L453-L467